### PR TITLE
General report shows only number of affected genes for dismissed SVs

### LINKED ISSUE #2511: Shorten dismissed SV variants on general report
Dismissed variants on a general report is too verbose. It would be ok to remove e.g. genes and just show number of genes if the number is large.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2511 

**How to test**:
1. Dismiss a number of structural variants for a case (big variants encompassing many genes are especially good for testing)
2. Go to general report and check the different between this branch and master
3. Dismissed SNVs should look exactly the same as in master branch.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Before:

image

After:

image

**Comment by northwestwitch** [comment]
Ready for review. I don't know for how long, but it's also on clinical-db stage now

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 1]
```diff
@@ -748,15 +748,15 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-secondary">Pa
               <td width="5%">#{{loop.index}}</td>
               {% if variant.category == 'snv' %}
                 <td width="25%"><a href="{{ url_for('variant.variant', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" target="_blank"><strong>{{variant.display_name[:30]}}</strong></a></td>
-              {% else %}
+                <td width="25%">
+    	             {% for gene in variant.genes %}
+                    <a href="http://www.genenames.org/cgi-bin/gene_symbol_report?match={{ gene.hgnc_symbol }}" target="_blank">{{ gene.hgnc_symbol }}&nbsp;<span class="badge badge-secondary">{{gene.region_annotation}}</span></a><br>
+                   {% endfor %}
+                </td>
+              {% else %} <!-- SVs -->
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:756]: How about other categories, like cancer and str? Cancer will go nicely with snv (and cancer_sv obviously with sv). Str could use a gene or str_repid if none is set (older cases).

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 2]
```diff
@@ -748,15 +748,15 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-secondary">Pa
...
                 <td width="25%"><a href="{{ url_for('variant.sv_variant', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" target="_blank"><strong>chr{{variant.chromosome}}:{{variant.position}}_{{variant.sub_category|upper}}</strong></a></td>
+                <td width="25%">{{ variant.genes|length }}</td>
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:758]: A conditional if that is below like three genes perhaps, then write them, else say N genes.

**Review by dnil — APPROVED** [review]
Minor revision.. 😸

File: scout/server/blueprints/cases/templates/cases/case_report.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case_report.html:756]: Cancer SVs and SNVs will look like RD SVs and SNVs. I'll fix strs as you suggested!

File: scout/server/blueprints/cases/templates/cases/case_report.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case_report.html:758]: Yeah, why not?

**Comment by northwestwitch** [comment]
Changed it like this:

image

It's on stage now: https://scout-stage.scilifelab.se/cust000/643594-300M

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 3]
```diff
@@ -738,143 +750,147 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-secondary">Pa
...
+                  <a href="{{ url_for('variant.sv_variant', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" target="_blank"><strong>chr{{variant.chromosome}}:{{variant.position}}_{{variant.sub_category|upper}}</strong></a>
+                {% else %} <!--strs -->
+                  <a href="{{ url_for('variant.variant', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" target="_blank"><strong>rep. {{variant.str_repid}}</strong></a>
+                {% endif %}
+              </td>
+              <td width="5%">{{variant.category}}</td>
+              <td width="15%">
+                {{ dismissed_gene_list(variant) }}
               </td>
-          <td>
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case_report.html:760]: I've just moved the indent of this block, no changes made to the code

**Pull request merged**